siliconcompiler/project.py

```
@@ -1273,7 +1273,8 @@ def snapshot(self, path: str = None, jobname: str = None, display: bool = True)
             _open_summary_image(path)
 
     def show(self, filename: Optional[str] = None, screenshot: bool = False,
-             extension: Optional[str] = None, tool: Optional[str] = None) -> str:
+             extension: Optional[str] = None, tool: Optional[str] = None,
+             open: bool = False) -> str:
```

Ruff (0.15.12)

[error] Function argument `open` is shadowing a Python builtin

(A002)